packages/elements/src/react/sign-in/root.tsx

```diff
@@ -14,7 +14,7 @@ import { Router, useClerkRouter, useVirtualRouter } from '~/react/router';
 import { SignInRouterCtx } from '~/react/sign-in/context';
 
 import { Form } from '../common/form';
-import { usePathnameWithoutCatchAll } from '../utils/path-inference/next';
+import { removeOptionalCatchAllSegment } from '../utils/path-inference/utils';
```

Comment:
The previous version was Next specific, which required that Elements depend on Next.js, resulting in it being included in the `clerk-js` bundle. This change instead uses the host router without specifically calling `useRouter`.